draft-ietf-mls-architecture.md

```diff
@@ -296,13 +251,13 @@ public values such as a name (an identity), a public encryption key
 and a public signature key. Ownership of a client by a user is
 determined by the fact that the user has knowledge of the
 associated secret values. When a client is part of a Group, it is
-called a Member and its signature key pair uniquely defines its
-identity to other clients or members in the Group.
+called a Member and its signature key pair uniquely identifies it
+to other clients or members in the Group.
 In some messaging systems, clients belonging to the same user must
-all share the same identity key pair, but MLS does not assume this.
+all share the same signature key pair, but MLS does not assume this.
 
-Users will typically own multiple clients, potentially one or more per
-end-user devices (phones, web clients or other devices...) and may
+Users will often use multiple clients, potentially one or more per
+device (phones, web clients or other devices...), and may
 choose to authenticate using the same signature key across devices,
```

Comment:
While you're at it, I believe it's not really possible anymore to use the same signature key across client. At least not if two client are meant to be members of the same group. The application could choose to authenticate the credentials based on some other key material, but that really depends on how the AS is designed. I would suggest we discuss the different options in the section describing the AS instead.

Reply:
Fair point, will try to reword.